src/main/kotlin/com/michaelsgroi/baseballreference/BrWarDailyLines.kt

```diff
@@ -2,67 +2,66 @@ package com.michaelsgroi.baseballreference
 
```

Comment:
not really a renaming; `BrWarDaily(Bat|Pitch)Lines`—2 very similar files—replaced with `BrWarDailyLines`